src/grid-list/grid-list.jsx

```diff
@@ -105,16 +102,16 @@ const GridList = React.createClass({
       }
       const childCols = currentChild.props.cols || 1;
       const childRows = currentChild.props.rows || 1;
-      const itemStyle = this.mergeStyles(styles.item, {
+      const itemStyle = Object.assign({}, styles.item, {
```

Comment:
I see, well for now, I would say let's keep our migration safe. We will tweak and improve stuff later.